core/serv/src/main.rs

```diff
@@ -390,14 +390,17 @@ impl ServiceCommand {
                     }),
                     &vec![
                         ("actix_http::response", log::LevelFilter::Off),
-                        ("h2", log::LevelFilter::Info),
+                        ("h2", log::LevelFilter::Off),
                         ("hyper", log::LevelFilter::Info),
                         ("reqwest", log::LevelFilter::Info),
                         ("tokio_core", log::LevelFilter::Info),
                         ("tokio_reactor", log::LevelFilter::Info),
                         ("trust_dns_resolver", log::LevelFilter::Info),
                         ("trust_dns_proto", log::LevelFilter::Info),
                         ("web3", log::LevelFilter::Info),
+                        ("tokio_util", log::LevelFilter::Off),
+                        ("mio", log::LevelFilter::Off),
+                        ("ya_persistence::executor", log::LevelFilter::Debug),
```

Comment:
always on debug?

Reply:
They were really annoying, but maybe I will remove this

Reply:
shouldnt they be on info / warn then?

Reply:
i see you removed it :) will resolve the comment